x-pack/legacy/plugins/epm/server/lib/elasticsearch/ingest_pipeline/ingest_pipelines.ts

```diff
@@ -57,12 +57,12 @@ export async function installPipelinesForDataset({
 
   pipelinePaths.forEach(path => {
     const { name, extension } = getNameAndExtension(path);
-    const nameForInstallation = getPipelineNameForInstallation(
-      name,
+    const nameForInstallation = getPipelineNameForInstallation({
+      pipelineName: name,
       dataset,
       datasourceName,
```

Comment:
Here we pass `datasourceName` as argument 3, and `packageName` as argument 4, matching the function signature at https://github.com/elastic/kibana/pull/53848/files#diff-745e02925487aa37c44700d53aa633c9L153